docs/modules/ROOT/pages/vshn-managed/postgresql/connect.adoc

```diff
@@ -0,0 +1,50 @@
+== How to connect to Your instance ==
+
+=== Problem
+
+You want to either connect to Your PostgreSQL cluster from outside of the cluster, or You need to port-forward, so You can use Your favorite database management tool.
```

Comment:
```suggestion
You want to either connect to your PostgreSQL cluster from outside of the cluster or need to port-forward so that you can use your favorite database management tool.
```